lisp/ess-r-flymake.el

```diff
@@ -62,24 +62,44 @@ each element is passed as argument to 'lintr::with_defaults'."
 
 (defvar-local ess-r--flymake-proc nil)
 
+(defvar-local ess-r--lintr-file nil
+  "Location of the .lintr file for this buffer.")
+
 (defvar ess-r--flymake-def-linter
   (replace-regexp-in-string
    "[\n\t ]+" " "
-   "esslint <- function(str, linters, cache) {
```

Comment:
Shall we move this to etc/ESSR on this occasion?

Reply:
If you want, sure. It's convenient to have it here, though.

I really don't have a strong preference though.

Reply:
I don't think it's that convenient, it's just a string not a real code. We will eventually be making ESSR a package and would be nice to track all the relevant R code there.  Let's keep an uniform convention throughout ESS.

Reply:
As it is the R process we're spawning for flymake doesn't load ESSR though. Is there anywhere else in ESS that spawns background R processes quickly then exits? 

I don't think we even need to define `esslint`, since flymake will log to the *Flymake-log* buffer if anything goes wrong anyway.

Reply:
I missed this part. Indeed, it's a different beast in here.